src/QirRuntime/lib/QIR/intrinsicsOut.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <iostream>
+
+#include "qirTypes.hpp"
+#include "quantum__qis.hpp"
+
+// Forward declarations:
+static std::ostream& GetQOstream();
```

Comment:
One more small request. Since the public API is named `SetOutputStream` now, it would be great to match the names in the internal implementation with the public API name (i.e. `GetOutputStream`, `currOutputStream`).

Reply:
Fixed